python_cross_site_scripting/flowsn_proplookup/1.py

```
@@ -0,0 +1,256 @@
+          container.textContent = '(No summary available)';
+        }
+      });
+    </script>
+  {% endif %}
+</body>
+</html>
+'''
+
+if __name__ == '__main__':
+    port = int(os.environ.get('PORT', 5000))
+    app.run(host='0.0.0.0', port=port, debug=True)
```

Ruff (0.13.3)

Possible binding to all interfaces

(S104)